fn main() -> Result<(), anyhow::Error> {
  thunk::thunk();

  Ok(())
}
